src/reactpy_django/templatetags/jinja.py

```diff
@@ -0,0 +1,67 @@
+# Copyright © 2023 Innovatie Ltd. All rights reserved.
+"""
+Jinja support.
+"""
+import typing as t
+
+from django.template import RequestContext, loader
+from jinja2 import pass_context
+from jinja2.ext import Extension
+from jinja2.runtime import Context, Undefined
+
+from .reactpy import component as djt_component
+from .. import config
+
+
+class ReactPyExtension(Extension):
+    """
+    Jinja has more expressive power than core Django's templates, and can
+    directly handle expansions such as:
+
+        {{ component(*args, **kwargs) }}
+    """
+    DJT_TEMPLATE = 'reactpy/component.html'
```

Comment:
No point as storing this as a class attribute if it's only used once

Reply:
This is really a reminder to pull this out into somewhere as thew value is required to be the same as used in the decorator for templatetags.component.

Reply:
@Archmonger I think this is the last remaining unresolved item. Any thoughts on sharing this value, or shall I just go ahead and hard code it?

Reply:
Let's make it a global within the Django template tag module for now.